### ISSUE TITLE
It would be great for many people to post here binaries for the fpm arm packages. Btw thank you so much for the solution :)

### ISSUE THREAD

**Issue by daic11** [jgresham/fpm-arm-binary#1]

**Comment by jgresham** [comment]
you're welcome! I'll try to support however I can. 

But I switched to `electron-forge` so I'm no longer using this or `electron-builder` because it seemed like `electron-builder` was falling behind.

**Comment by daic11** [comment]
> you're welcome! I'll try to support however I can. 
> 
> But I switched to `electron-forge` so I'm no longer using this or `electron-builder` because it seemed like `electron-builder` was falling behind.

So you mean if I'll change to electron-forge it'll be working with arm? Interesting thank uou, I'll try.

**Comment by jgresham** [comment]
@daic11  Yes, forge works with arm better from my experience. No windows arm (yet) iirc.
